Tests/ResilientDecodingTests/XCTestManifests.swift

```diff
@@ -18,7 +18,7 @@ extension ResilientArrayTests {
         ("testDecodesNullValue", testDecodesNullValue),
         ("testDecodesValidInputWithoutErrors", testDecodesValidInputWithoutErrors),
         ("testDecodesWhenMissingKeys", testDecodesWhenMissingKeys),
-        ("testResilientlyDecodesArratWithInvalidElements", testResilientlyDecodesArratWithInvalidElements),
+        ("testResilientlyDecodesArrayWithInvalidElements", testResilientlyDecodesArrayWithInvalidElements),
```

Comment:
Do we actually run these Linux tests? Asking to learn.

Reply:
Yup! https://github.com/airbnb/ResilientDecoding/pull/3